### ISSUE TITLE
Unsupported Install Addon URL

### ISSUE THREAD

**Issue by drejjmit** [jliddev/WowUp#132]
https://www.curseforge.com/wow/addons/bug-grabber
"Addon not found"

**Comment by mkvlrn** [comment]
URLs should be github repos, I think. Not curse project pages.

![image](https://user-images.githubusercontent.com/6718750/95978792-0b222a80-0df1-11eb-8783-01c81375b1c2.png)

**Comment by drejjmit** [comment]
Other Curse links does work even tho it says GitHub only.

**Comment by drejjmit** [comment]
https://www.curseforge.com/wow/addons/postal

**Comment by orielehs** [comment]
I was able to do the same with https://www.curseforge.com/wow/addons/azeritetooltip , and I've had github URLs not accepted. I'm assuming curseforge uses github as part of its backend.

**Comment by Urganot** [comment]
I found the reason for the behavior. When the app searches for an addon, it uses the slug. when the api returns its results, it uses the addons name to filter. so for example:
AddonName: TradeSkillMaster
Slug: tradeskill-master

Those two wont match. Thats the reason curse wont return the correct addon. Maybe there is the possibility to sreach via slug?
In the meantime I improved this behavior in my pullrequest #182
